Missing 2008 R2 in system requirements notes (#6746)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/onboard-downlevel.md

- Install the [February 2018 monthly update rollup](https://support.microsoft.com/help/4074598/windows-7-update-kb4074598)

  > [!NOTE]

>   > Only applicable for Windows Server 2008 R2, Windows 7 SP1 Enterprise and Windows 7 SP1 Pro.

Reviewer: ```suggestion
  > Only applicable for Windows Server 2008 R2, Windows 7 SP1 Enterprise, and Windows 7 SP1 Pro.
```

- Install the [Update for customer experience and diagnostic telemetry](https://support.microsoft.com/help/3080149/update-for-customer-experience-and-diagnostic-telemetry)

- Install either [.NET framework 4.5](https://www.microsoft.com/download/details.aspx?id=30653) (or later) or [KB3154518](https://support.microsoft.com/help/3154518/support-for-tls-system-default-versions-included-in-the-net-framework)

    > [!NOTE]

>     > Only applicable for Windows Server 2008 R2, Windows 7 SP1 Enterprise and Windows 7 SP1 Pro.

Reviewer: ```suggestion
    > Only applicable for Windows Server 2008 R2, Windows 7 SP1 Enterprise, and Windows 7 SP1 Pro.
```

    > Don't install .NET Framework 4.0.x, since it will negate the above installation.

>     > Attention, installation of .NET 4.5 might require a reboot after installation

Reviewer: ```suggestion
    > Only applicable for Windows Server 2008 R2, Windows 7 SP1 Enterprise, and Windows 7 SP1 Pro.
    >
    > Don't install .NET Framework 4.0.x, since it will negate the above installation.
    >
    > Installation of .NET 4.5 might require you to restart your computer after installation.
```
